<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToeBoard</class>
 <widget class="QWidget" name="TicTacToeBoard">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="characterButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>26</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose character</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numberButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose player number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>98</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton6">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>98</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>22</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton9">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>174</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>22</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton8">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>174</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton7">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>174</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>98</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>22</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton1</tabstop>
  <tabstop>pushButton2</tabstop>
  <tabstop>pushButton3</tabstop>
  <tabstop>pushButton4</tabstop>
  <tabstop>pushButton5</tabstop>
  <tabstop>pushButton6</tabstop>
  <tabstop>pushButton7</tabstop>
  <tabstop>pushButton8</tabstop>
  <tabstop>pushButton9</tabstop>
  <tabstop>characterButton</tabstop>
  <tabstop>numberButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
